sklearn/manifold/_mds.py

```diff
@@ -152,6 +172,7 @@ def smacof(
     eps=1e-3,
     random_state=None,
     return_n_iter=False,
+    normalized_stress=False,
```

Comment:
Can we extend the docstring for `eps` in `smacof` the same way you did for `MDS`?